src/CommandHandler.h

```diff
@@ -7,6 +7,12 @@
 #include "Msg.h"
 #include <memory>
 
+// class JobIdException;
```

Comment:
This comment just repeats the line below ;)
I think we can lose it

Reply:
Right.. I wanted to add a couple of lines of documentation but I forgot. I'll an update soon.